src/bidiMapper/domains/context/browsingContextImpl.ts

```diff
@@ -291,17 +291,11 @@ export class BrowsingContextImpl {
     };
   }
 
-  #initListeners() {
-    this.#cdpTarget.cdpClient.on(
-      'Target.targetInfoChanged',
-      (params: Protocol.Target.TargetInfoChangedEvent) => {
-        if (this.id !== params.targetInfo.targetId) {
-          return;
-        }
-        this.#url = params.targetInfo.url;
-      }
-    );
+  targetInfoChanged(params: Protocol.Target.TargetInfoChangedEvent) {
```

Comment:
update this method name to a verb phrase, maybe "updateTargetInfoUrl". Otherwise it reads like an event